src/update/UpdateManager.cpp

```diff
@@ -123,29 +119,45 @@ void UpdateManager::OnTimer(wxTimerEvent& WXUNUSED(event))
     if (updatesCheckingEnabled && IsTimeForUpdatesChecking())
         GetUpdates();
 
-    mTimer.StartOnce(mUpdateCheckingInterval);
+    mTimer.StartOnce(std::chrono::duration_cast<std::chrono::milliseconds>(
```

Comment:
I think ADL would let you omit the name qualification :-)

Reply:
one of them at least

Reply:
Visual Studio is not very happy :(